tools/loader.md

```diff
@@ -7,13 +7,13 @@ category: advanced
 
 ## What is Loader?
 
-Developed by PingCAP, Loader is a data import tool and it imports data to TiDB and MySQL.
+Developed by PingCAP, Loader is a data import tool and it imports data to TiDB 
```

Comment:
Loader is a data import tool and it imports data to TiDB -> 
Loader is a data import tool to load data to TiDB.